deployments/kustomize/overlays/openshift/images/image-builds/app.yaml

```diff
@@ -36,7 +36,17 @@ spec:
     type: Docker
   source:
     contextDir: frontend
-    dockerfile: ""
+    dockerfile: |-
+      FROM node:18.14.0-alpine3.17
+      WORKDIR /app
+      COPY public/ public/
+      COPY src/ src/
+      COPY package.json .
+      ADD tsconfig.json .
+      ADD .eslintrc.js .
```

Comment:
Looks like the ticket just requires you to copy a Dockerfile into app.yaml config so looks good there. Is there a reason linting and pretty-print spec files are included? Do these developer utilities get run in the Openshift environment?

Reply:
that's a good question. I'm not sure they would be.